components/link/link.js

```diff
@@ -2,12 +2,13 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styles from './link.module.scss';
 
-const Link = ({ href, children }) => (
-  <a className={styles.link} href={href}>{children}</a>
+const Link = ({ href, name, children }) => (
+  <a className={styles.link} href={href} aria-label={`${name}'s ${children}'`}>{children}</a>
```

Comment:
Since this is a generic `Link` component, as opposed to something with a more specific purpose (eg a single  `SocialLink` component), I don't think that `name` is the appropriate level of abstraction here. I think the prop should just be ` aria-label`, and the current logic for formatting that string lifted from the component into the parent. 

In this case, it looks like this is only used in the context of `SocialLinks`, so I think the string formatting can go directly in that component. Hypothetically, if we did need to reuse a component that included the `name` formatting, we could use composition: a `SocialLink` component which accepts the `name`, then passes the formatted name to the underlying generic `Link` component.

Reply:
Sounds good to me